rocketpy/simulation/monte_carlo.py

```diff
@@ -654,9 +682,15 @@ def set_processed_results(self):
         """
         self.processed_results = {}
         for result, values in self.results.items():
-            mean = np.mean(values)
-            stdev = np.std(values)
-            self.processed_results[result] = (mean, stdev)
+            try:
+                if isinstance(values[0], float):
```

Comment:
I think checking for float is a bit dangerous here.
Isn't there any other types that would need to be included here? @phmbressan what do you think about it?

Btw, @Lucas-Prates for future PRs, we may want to start considering other statistics as well, such as median, min, max, etc.

Reply:
@Lucas-Prates can you open up an issue so we tackle that point later? Thx